stok.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const Hapi = require('hapi');
+
+class Stok {
+  constructor(options) {
+    this.options = options;
+  }
+
+  createServer(routes) {
+    return new Promise((resolve, reject) => {
+      const server = new Hapi.Server();
+      server.connection({ port: this.options.web.port });
+
+      // Monkey patch the route method and add error handling to
+      // all of our routes
+      server.route = ((original) => {
+        routes => {
+          routes.forEach(route => {
```

Comment:
`routes` may be a single route or an array of routes, so you need to handle both cases. The easiest way would be to just do:

```js
if (!Array.isArray(routes)) {
  routes = [routes];
}
```